test/system/questions_test.rb

```diff
@@ -257,4 +257,14 @@ class QuestionsTest < ApplicationSystemTestCase
     end
     assert_text 'Watch中'
   end
+
+  test 'show number of comments' do
+    visit_with_auth questions_path, 'kimura'
+    assert_text questions(:question14).title
```

Comment:
```suggestion
    assert_text 'コメント数表示テスト用の質問'
```

下のコードではベタ書きしてるので合わせた方が一貫性の意味で良いと思います。

Reply:
書いてくださったとおり、「一貫性」の視点がありませんでした。
ベタ書きに修正しました。
ありがとうございます！